pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -837,7 +838,8 @@ func NewPool(args *NewPoolArgs) *appsv1.StatefulSet {
 			UpdateStrategy: appsv1.StatefulSetUpdateStrategy{
 				Type: miniov2.DefaultUpdateStrategy,
 				RollingUpdate: &appsv1.RollingUpdateStatefulSetStrategy{
-					MaxUnavailable: &unavailable,
```

Comment:
remove the field `RollingUpdate`

Reply:
I verified that removing `UpdateStrategy` also solves this issue hence pushed another commit.

Reply:
Removing the entire `UpdateStrategy` is not equivalent to the previous implementation when `MaxUnavailable` is introduced to the operator in https://github.com/minio/operator/pull/1290. Are you sure this is OK so far?

Reply:
@chitoku-k I think so. We will create PDB now.